client/pages/live-map.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { Map, GoogleApiWrapper, Marker } from 'google-maps-react';
+
+class LiveMap extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      dailyRoutes: [],
+      maxIncrease: 0,
+      totalVehicles: 0,
+      count: 0,
+      mapCenter: {
+        lat: 25.774,
+        lng: -80.190
+      },
+      vehicles: [{
+        vehicleId: 1,
+        coords: {
+          lat: 25.774,
+          lng: -80.190
+        }
+      }, {
+        vehicleId: 2,
+        coords: {
+          lat: 24.774,
+          lng: -80.190
+        }
+      }]
+    };
+    fetch('/api/dailyroutes')
+      .then(res => res.json())
+      .then(result => {
+
+        let routesArr = [];
+        const dailyRoutesArr = [];
+        this.setState({ totalVehicles: result.rows.length });
```

Comment:
is `totalVehicles` the number of vehicles in your state? if that's the case, you don't need to store `totalVehicles`. Instead just look at `this.state.vehicles.length`.